example_configs/minio.md

```diff
@@ -0,0 +1,37 @@
+# MinIO Configuration
+
+MinIO is a High-Performance Object Storage released under GNU Affero General Public License v3. 0. It is API compatible with the Amazon S3 cloud storage service. This example assists with basic LDAP configuration and policy attachment.
+
+## LDAP Config
+
+### Navigation
+
+- Login to the WebUI as a consoleAdmin user
+- Navigate to `Administrator > Identity > LDAP`
+- Click `Edit Configuration`
+
+### Configuration Options
+
+- Server Insecure: `Enabled`
+- Server Address: `Hostname or IP for your LLDAP host`
```

Comment:
Small note: I would put the values between backticks when it's a literal value (such as `Enabled` or `uid=admin`), but not when it's a description of the value

Reply:
I think you missed the "Hostname or IP" and the password description.